pages/guideforms.py

```diff
@@ -400,10 +400,10 @@ def __init__(self, attrs=None):
          'https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/public/common/features.h'
          '" target="_blank">examples here</a>) that can '
          'be flipped off in case of compat issues</li>'
-         '<li>Consider reaching out to android-webview-dev@chromium.org for '
+         '<li>Consider contacting android-webview-dev@chromium.org for '
          'advice</li>'
-         '<li>If you are not sure, just put "not sure" as the answer here and '
-         'the API owners can help during the review of your intent-to-ship</li>'
+         '<li>If you are not sure, just put "not sure" as the answer and '
+         'the API owners can help during the review of your intent to ship</li>'
```

Comment:
Should this be "Intent to Ship", or should "Intent to Extend Origin Trial" (and other similar instances) be lower cased?

Reply:
See #2007 for my PR.  

 * I did change "intent to ship" to "Intent to Ship" since all the other cases do capitalize.
 * The `\n\n` chars need to be `<br/><br/>`, or else the preceding text needs to be in a block.
 * I am only migrating the help_text so far, so the label text is unchanged by my PR.

Reply:
I should have fixed the capitalization of intents. Common nouns are like classes. Proper nouns are like instances. They're not exact parallels, but they're close. Common nouns are not capitalized. The intent references in the instructions are common nouns even though they might seem to refer to a single intent. If I say 'your city', I don't capitalize it just because 'city' stands in for 'Seattle'. I've fixed this throughout.

I've fixed the line breaks.